lib/pipedrive/base.rb

```diff
@@ -15,8 +15,8 @@ module Pipedrive
   class Base < OpenStruct
 
     include HTTParty
-
-    base_uri 'https://api.pipedrive.com/v1'
+    
+    base_uri 'api.pipedrive.com/v1'
```

Comment:
You sure that this would work without the `https`?

Reply:
that's why we have the logic in the initialiser. IMO the library is supposed to work like this and everyone that prefers https over http could patch it the way you've done

Reply:
Now, I saw your other comment from the other PR. If Pipedrive doesn't work with http, then yes, it is better to make it https here instead of patching it in the Monolith